atlasdb-config/build.gradle

```diff
@@ -18,7 +18,9 @@ dependencies {
     compile group: 'com.palantir.remoting3', name: 'keystores'
     compile group: 'com.palantir.safe-logging', name: 'safe-logging'
     compile (group: 'com.palantir.tritium', name: 'tritium-lib') {
+        exclude (group: 'com.palantir.remoting3', module: 'tracing')
```

Comment:
If we dont re-add the excludes, how will it be avaialble to the module. Possibly they are coming in transitively but adding it explicitly feels like a safer option, especially as some classes might be used at runtime only?

Reply:
There are multiple transitive dependencies, which bring those modules in. I will add anyways to be explicit. Shouldn't be a problem as long as we don't specify the version.